### PR TITLE
feat(cluster): add redeploy banner

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-747
+ And update Banner component with CVA

Capture d’écran 2023-09-26 à 13 52 16

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Write comments that add new information that doesn’t exist in the code

**Review by ctjhoa — CHANGES_REQUESTED** [review]

File: libs/shared/ui/src/lib/components/banner/banner.tsx [hunk 1]
```diff
@@ -1,56 +1,55 @@
...
-
+export const Banner = forwardRef<HTMLDivElement, PropsWithChildren<BannerProps>>(function Banner(
+  { children, buttonLabel, buttonIconRight, onClickButton, color = 'yellow' },
+  forwardedRef
+) {
   return (
-    <div data-testid="banner" className={`${classNameStyle || ''}`}>
-      <div className="flex h-10 items-center justify-center font-medium text-sm">
+    <div className={bannerVariants({ color })}>
+      <div className="flex h-10 items-center justify-center font-medium text-sm" ref={forwardedRef}>
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/shared/ui/src/lib/components/banner/banner.tsx]: https://github.com/Qovery/console/pull/856/commits/226de489823d5c5529a0e989c6b810b291004fe8

File: libs/shared/ui/src/lib/components/banner/banner.stories.tsx [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 export default {
   component: Banner,
   title: 'Banner',
   argTypes: {
-    bannerStyle: {
+    color: {
       control: {
         type: 'select',
-        options: BannerStyle,
+        options: ['yellow', 'brand'],
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/shared/ui/src/lib/components/banner/banner.stories.tsx]: https://github.com/Qovery/console/pull/856/commits/226de489823d5c5529a0e989c6b810b291004fe8

File: libs/shared/ui/src/lib/components/banner/banner.tsx [hunk 3]
```diff
@@ -33,21 +34,22 @@ export const Banner = forwardRef<HTMLDivElement, PropsWithChildren<BannerProps>>
...
-            className={buttonVariants({ color })}
-            iconRight={buttonIconRight}
-            onClick={onClickButton}
-          >
-            {buttonLabel}
-          </Button>
-        )}
-      </div>
+    <div
+      className={twMerge('flex h-10 items-center justify-center font-medium text-sm', bannerVariants({ color }))}
```
Comment (ctjhoa) [inline libs/shared/ui/src/lib/components/banner/banner.tsx:38]: move `'flex h-10 items-center justify-center font-medium text-sm'`  into `bannerVariants` so you can get rid of `twMerge`

Comment (RemiBonnet) [reply libs/shared/ui/src/lib/components/banner/banner.tsx]: fixed https://github.com/Qovery/console/pull/856/commits/0f232ffdc01d3a3f5b5618f00a0be5989e20aacf

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**